docs/s3.md

```diff
@@ -34,55 +34,63 @@ leads to an HTML or .txt file, it will get rendered.  And if it leads to any oth
 
 ## Developer access/use
 
-The bucket is owned by the Developer user for our ESO account which we should all have access
-to.  Use that from the aws console to manipulate anything about how the bucket is setup. 
+The bucket is owned by the Developer role for our ESO account which we should all have access
+to.  Use that from the aws console to manipulate anything about how the bucket is setup.
 
 ### Install the aws cli
 
-Follow the 
-[Installing AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html) 
+Follow the
+[Installing AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html)
 instructions for the machine you want to access s3 from.
 
 ### Credentials
 
-IT set up a service account for us to use when accessing the bucket via the cli.  They
-added the credentials to the 1password SLV team vault as the S3 service account login.
- 
+Use the puppet-bastion account with the eso profile to connect provide aws cli
+access and assume the proper role.  See [confluence
+docs](https://confluence.puppetlabs.com/pages/viewpage.action?pageId=104761111#AmazonWebServices\(AWS\)-Assumerolesviathecommandline)
+for reference.
 
-Create a `.aws/credentials` file in your homedir.
-Something like this:
+Create a `$HOME/.aws/credentials` file containing at least a default section.
 ```
 [default]
 aws_access_key_id = <personal id>
 aws_secret_access_key = <personal key>
-          
-[slv_s3_service_account]
-aws_access_key_id = <slv s3 id>
-aws_secret_access_key = <slv s3 key>
+```
+
+Create a `$HOME/.aws/config` file containing at least a default and eso profile section.
+```
+[default]
+output = json
+region = us-west-2
+
+[profile eso]
+role_arn = arn:aws:iam::<eso-account-number>:role/Developer
+mfa_serial = arn:aws:iam::103716600232:mfa/<username>
```

Comment:
@billclaytor are the numbers for your MFA the same as this?  Just the username is different?

Reply:
The mfa appears to be universal.